example/datatypes.card.js

```diff
@@ -72,6 +72,11 @@ var ListToggle = React.createClass({
   }
 });
 
+devcard('Set',
```

Comment:
Is https://github.com/glenjamin/devcards-js/blob/master/example/.babelrc enough to guarantee this'll be fine?